### DISCUSSION TITLE
weekly/2024-06-16

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#249] [Announcements]
# weekly/2024-06-16

① Bread Jam - 变量上色 ② JsonTree.js - JSON 数据可视化 ③ Reshot - 免费图标和插图 ④ 2024 年度备受期待的 JavaScript 项目 ⑤ 万能视频图片解析下载 ⑥ Chromicons - 线条风格图标集合 ⑦ DGM.js - 无限画布 ⑧ 英：从头开始了解 JS Promise ⑨  TextToSpeech - 免费在线转换文字为语音

https://sugarat.top/weekly/2024-06-16.html

**Comment by iygxv** [comment]
怎么感觉少了表情呢？

**Reply by ATQQ** [reply]
少什么表情

**Reply by iygxv** [reply]
想给你回复表情哈哈
